src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -570,6 +570,21 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
+  const renderShowZendeskToggle = () => (
+    <div className="toggle-room">
+      <h4 className="italic input-header">Show Zendesk support popup</h4>
+      <label id={"showZendesk"} className="switch">
+        <input
+          type="checkbox"
+          id={"showZendesk"}
+          name={"showZendesk"}
+          ref={register}
+        />
+        <span className="slider round"></span>
+      </label>
+    </div>
+  );
```

Comment:
This should be a standalone component rather than a render function. The strings in the JSX don't need the `{}` around them either, as you can just use the string directly.

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/101